packages/kbn-test/src/functional_tests/lib/run_kibana_server.js

```diff
@@ -31,7 +31,7 @@ export async function runKibanaServer({ procs, config, options }) {
       ...process.env,
     },
     cwd: installDir || KIBANA_ROOT,
-    wait: /Server running/,
+    wait: /The core is running/,
```

Comment:
__concern:__ I'm a bit concerned about this change cause optimizer and cluster manager will log this message as well and hence test runner will think that Kibana is ready before it's actually ready. `Server running` was issued only by the "server" process if I'm not mistaken. I believe we should address it before we can merge it. The simplest fix I can think of is to log this message only if `HttpService.shouldListen` (that excludes Cluster manager and optimizer processes) is `true` or something like this.

Reply:
fixed